src/ColumnsSummary.php

```diff
@@ -105,12 +90,7 @@ public function render(string $key): ?string
 			return '';
 		}
 
-		return number_format(
-			$this->summary[$key],
-			$this->format[$key][0],
-			$this->format[$key][1],
-			$this->format[$key][2]
-		);
+		return number_format($this->summary[$key], $this->format[$key][0], $this->format[$key][1], $this->format[$key][2]);
```

Comment:
Please revert this line - Having 4 function arguments on line line may be not well readable for some people. Thanks 👍

Reply:
Solved